gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -38,10 +32,10 @@ def __init__(self, tag: str) -> None:
         argparser: argparse.ArgumentParser = self.construct_argparser()
         self.parsed_args: argparse.Namespace = argparser.parse_args()
         if self.parsed_args.dask_tmpdir:
-            dask.config.set({'temporary_directory': DEFAULT_OUTPUT_ROOT / self.parsed_args.dask_tmpdir})
+            dask.config.set({"temporary_directory": DEFAULT_OUTPUT_ROOT / self.parsed_args.dask_tmpdir})
 
         self.loader: LoaderBase = self.construct_loader()
-        self.retriever = self.construct_retriever()
+        # self.retriever = self.construct_retriever()
```

Comment:
should we remove this commented out line?

Reply:
gentle bump on this : - )

Reply:
Done